src/celeritas/optical/ScintillationData.hh

```diff
@@ -20,71 +20,73 @@ namespace optical
 {
 //---------------------------------------------------------------------------//
 /*!
- * Material dependent scintillation property.
+ * Parameterized scintillation properties.
  *
- * Components represent different scintillation emissions, such as
- * prompt/fast, intermediate, and slow. They can be material-only or depend on
- * the incident particle type.
+ * This component represents one type of scintillation emissions, such as
+ * prompt/fast, intermediate, or slow. It can be specific to a material or
+ * depend on the incident particle type.
  */
-struct ScintillationComponent
+struct ScintRecord
 {
-    real_type yield_frac{};  //!< Fraction of total yield (yield/sum(yields))
     real_type lambda_mean{};  //!< Mean wavelength
-    real_type lambda_sigma{};  //!< Standard dev. of wavelength
+    real_type lambda_sigma{};  //!< Standard deviation of wavelength
     real_type rise_time{};  //!< Rise time
     real_type fall_time{};  //!< Decay time
 
     //! Whether all data are assigned and valid
     explicit CELER_FUNCTION operator bool() const
     {
-        return yield_frac > 0 && yield_frac <= 1 && lambda_mean > 0
-               && lambda_sigma > 0 && rise_time >= 0 && fall_time > 0;
+        return lambda_mean > 0 && lambda_sigma >= 0 && rise_time >= 0
```

Comment:
Did you mean to change this to `lambda_sigma >= 0` (the normal distribution requires a positive stddev, and in the params construction we still validate that it's positive)?

Reply:
Oh I figured that for testing a delta function could be useful, but I didn't think to check that it would work with the normal distribution. Perhaps we should allow that as a degenerate case too?

Reply:
Yeah, I know we'd considered that on a couple past PRs but held off: https://github.com/celeritas-project/celeritas/pull/1108#discussion_r1491233708, https://github.com/celeritas-project/celeritas/pull/1086#discussion_r1462255701

Reply:
Wow, history repeats itself... thanks for the reminder.

Reply:
I've reverted this, as usual we can add "=0" when we encounter a use case 🤪